coreblocks/fu/fu_decoder.py

```diff
@@ -55,62 +53,54 @@ class DecoderManager:
     """
     Fn: Type[IntFlag]
```

Comment:
Sphinx doesn't generate documentation for this variable when it's comment block is placed above it.

Move it below. Then it generates documentation from the comment.

```suggestion
    """
    Type[IntFlag]

    Enumeration of instructions implemented in given functional unit.
    """
    Fn: Type[IntFlag]
```

Reply:
I think that `Type[IntFlag]` in comment blocks is not even needed since we have type hint.
Just 
```python
Fn: Type[IntFlag]
    """Enumeration of instructions implemented in given functional unit."""
```
should be enough (works for me).